### ISSUE TITLE
option to select all source

### ISSUE THREAD

**Issue by rosehartman** [InteragencyEcologicalProgram/discretewq#39]
I was pulling a bunch of data and it would be nice to have Source = "ALL" or something to pull all the potential sources rather than having to list them all. I can work on this (I think).

**Comment by sbashevkin** [comment]
That's how the package originally worked, but then when I was adding new data sources, it was changing the behavior of older code since the definition of "all" will change each time a new dataset is added. So I moved to forcing people to type out all the data sources they want, which is more future-proof (and makes people think about which data sources to use). But I'd be open to thoughts from you and @mountaindboz

**Comment by mountaindboz** [comment]
Yeah, I could go either way on this. I think we can write the `wq` to default to all sources and still have it be easy to update when adding new data sources. But I could also see the advantage of making people be explicit about what data sources they want to integrate. That being said, I have usually used `wq` to integrate all sources.

**Comment by rosehartman** [comment]
Hm. I would lean toward having it default to all sources, since that's how I usually use the package, but I can see both sides. I'll think about it.
